src/middlewares/Validate.js

```diff
@@ -258,6 +258,7 @@ class Validate {
     const day = d.getDate();
     const correctDate = new Date(year, month, day).toDateString();
     return [
+      check('tripId', 'tripId should be an integer').isNumeric(),
```

Comment:
isNumeric() method accepts integers, decimals and negative numbers. I suggest using isInt() method for validating only integers.